try:
    a=5/0
    prin(a)
except Exception as e :
    print(e)